nimbus.fml.yaml

```diff
@@ -165,61 +167,18 @@ features:
             EXPIRES_QUICKLY:
               priority: 100
               max-display-count: 1
-      - channel: developer
+      - channel: nightly
         value: {
           "messages": {
-            "my-viewpoint-survey": {
-              "title": "Message tile",
-              "text": "Love Firefox? Fill in our survey!",
-              "action": "https://surveyprovider.com/survey-id/{uuid}",
-              "trigger": [ "ALWAYS" ],
+            "default-browser-en": {
+              "text": "Set links from websites, emails, and messages to open automatically in Firefox.",
```

Comment:
We can use the existing strings:

```yaml
messages:
  default-browser:
    text:         default_browser_experiment_card_text
    button-label: preferences_set_as_default_browser 
```

and target everyone, not just the EN speakers.

Reply:
🙌🏽